README.md

```diff
@@ -30,6 +30,11 @@ A low-level library to play sound.
 
 ## Prerequisite
 
+On most platforms you will need a C/C++ compiler in your path that Go can use.
+
+- Windows: [MingW](https://www.mingw-w64.org/downloads/#mingw-builds) or similar
+- Mac/Linux: Should be installed by default, but if not try [GCC](https://gcc.gnu.org/) or [Clang](https://releases.llvm.org/download.html)
```

Comment:
macOS might not have gcc/clang. A user has to install them explicitly via Xcode tool. See https://ebiten.org/documents/install.html#C_compiler

Reply:
Windows doesn't require C compilers.

Reply:
Updated